test/fixtures_dynamodb_test.go

```diff
@@ -1,4 +1,4 @@
-//+build integration
+//+build integration,fixtures
```

Comment:
even though we need the fixtures for the test to succeed, i don't think this is a good idea. Because if we forget to add fixtures to the build flags then we don't even see this test failing

Reply:
same for following tests with the fixtures flag

Reply:
I think this is a good idea because by adding all the needed flags here, the IDE automatically adds them to the test run configuration, so you will never have to mark that stupid checkbox again: 

<img width="275" alt="image" src="https://user-images.githubusercontent.com/772593/78248739-79ef0480-74ed-11ea-9a84-64888e4ca845.png">

Reply:
i am not talking about local execution actually, but about CI. Someone who sets up a new ci process for a new repo might "forget" or just not know that you have to set these